### ISSUE TITLE
[PERF] Steamvr dropping frames regularly

### ISSUE THREAD

**Issue by The1AndMany** [ValveSoftware/SteamVR-for-Linux#261]
#### Your system information
* Steam client version (build number or date): Build date October 7, 2019 at 22:46:25
* Distribution (e.g. Ubuntu): Ubuntu LTS 18.04
* Graphics driver version (run nvidia-settings): NVIDIA 430.50
* Gist for SteamVR System Information: https://gist.github.com/The1AndMany/acb10f384f28fa86ca1d9cc2d80fa1ff
* Opted into Steam client beta?: [Yes/No] Yes
* Opted into SteamVR beta?: [Yes/No] Yes
* Have you checked for system updates?: [Yes/No] Yes

#### (Optional) Performance data:
The following fields are helpful in diagnosing performance issues:
* Gpuvis capture: (Can't figure out how to get gpuvis `make` to recognize libfreetype6)
* Perf top gist: (Unsure how to capture info)

#### Please describe your issue in as much detail as possible:
SteamVR has had issues for me since I got my Index. I've tried upgrading my graphics card and CPU, but I'm still having issues. Mainly, I get serious lag spikes in every game, and sometimes even in SteamVR Home. Worth noting is that these lag spikes also affect my computer's regular displays. The Advanced Frame Timing interface shows a spike in both CPU and GPU usage, but the graph freezes when the lag occurs, probably because it's not putting out any frames.

![Screenshot from 2019-10-10 20-19-14](https://user-images.githubusercontent.com/26637883/66623658-8da25980-eba1-11e9-8eaf-40e56f6b2788.png)

This makes games like Vacation Simulator or Accounting+ playable, if a bit headache-inducing, while games like Beat Saber and Audica are next to unplayable.

Willing to provide any additional information needed, but may need instructions.

**What should happen**
SteamVR acts normally, no performance issues

**What actually happens**
Frames drop every few seconds

#### Steps for reproducing this issue:
1. Load SteamVR
2. Dropped frames can occur in SteamVR Home, but are more common in games.

**Comment by SlickMcRunFast** [comment]
Same issues here. I tried the instructions for gpuvis and even tried nvidia's nsight, both just don't work. Seems only AMD users can capture traces. Maybe the instructions are outdated.

**Comment by KibaGasteiz** [comment]
This happens to me too, it's terrible, bought an Index in August and still can't use it. 1000$

**Comment by The1AndMany** [comment]
**_Update_**: After a while, it occurred to me that I still had subsampling at 20%, which was the setting I used with my old 1060 6GB. Disabling custom resolution and reverting the supersampling settings in Beat Saber and Audica fixed it, and now I'm playing both just fine. If I had to guess, I'd say the subsampling is CPU intensive, causing the massive spikes described by me and experienced by at least two others. This appears to have improved with a SteamVR update, but disabling these settings altogether is what helped the most.

@SlickMcRunFast & @KibaGasteiz, I'd recommend disabling the custom resolution. Assuming your GPU isn't being a bottleneck, this may fix things for you. Since I've solved my original problem, I'm gonna go ahead and close this issue. If things don't improve, consider opening a new ticket that specifically references the CPU spikes caused by subsampling.
